src/jobs/deploy-lambda.yml

```diff
@@ -1,9 +1,77 @@
 description: >
   This job deploys an application to aws lambda.
 
+parameters:
+  # Required Parameters
+  tag:
+    type: string
+    description: |
+      The semantic version of the project to be deployed
+
+  lambda-function-name:
+    type: string
+    description: |
+      The name of the lambda function to be updated.
+
+  lambda-package-value:
```

Comment:
instead of package value just have image-uri and zip-file so it makes more sense.